packages/paste-website/src/pages/components/display-heading/index.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: Display Heading
+package: '@twilio-paste/display-heading'
+description: Display heading is large, expressive text that stands out from traditional headings and is used for marketing and attention grabbing purposes.
+slug: /components/display-heading/
+---
+
+import {graphql} from 'gatsby';
+import {Card} from '@twilio-paste/card';
+import {DisplayHeading} from '@twilio-paste/display-heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {SidebarCategoryRoutes} from '../../../constants';
+import Changelog from '@twilio-paste/display-heading/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/display-heading"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(frontmatter: {slug: {eq: "/components/display-heading/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Display Heading"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/display-heading"
+  storybookUrl="/?path=/story/components-display-heading--all-variants"
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{DisplayHeading}} language="jsx">
+  {`<DisplayHeading as="h1" variant="displayHeading10" marginBottom="space0">
```

Comment:
Suggestion(blocking): this example should be `as="div"` because it lets people know immediately that this generally shouldnt be an actual heading.